src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,184 @@
+            break;
+        case COMMAND_EVENT_WORD:
+            if (!inputLine.contains(" /at ")) {
+                throw new IllegalEventInput();
+            }
+            taskCount = addEvent(tasksList, taskCount, inputLine);
+            break;
+        case COMMAND_LIST_WORD:
+            printTasks(tasksList, taskCount);
+            break;
+        case COMMAND_DONE_WORD:
+            if (!inputLine.contains(" ")) {
+                throw new EmptyTaskNumber();
+            }
+            markAsDone(tasksList, inputLine);
+            break;
+        default:
+            throw new IllegalCommand();
+        }
+        return taskCount;
+    }
+
+    public static void markAsDone(Task[] tasksList, String inputLine) {
+        int taskCompletedIndex = Integer.parseInt(inputLine.split("done ")[1]) - 1;
+        tasksList[taskCompletedIndex].setDone(true);
+        System.out.print("  Congratulations! You have completed the task:");
+        System.out.println(" " + tasksList[taskCompletedIndex]);
+    }
+
+    public static int addEvent(Task[] tasksList, int taskCount, String inputLine) {
+        String description = inputLine.split("event ")[1].split(" /at ")[0];
+        String date = inputLine.split(" /at ")[1];
+        tasksList[taskCount] = new Event(description, date);
+        System.out.println("  Ok, I've added this task:");
+        System.out.println("  " + tasksList[taskCount]);
+        taskCount++;
+        System.out.printf("  Now you have %d tasks in the list. Anything else?\n", taskCount);
+        return taskCount;
+    }
+
+    public static int addDeadline(Task[] tasksList, int taskCount, String inputLine) {
+        String description = inputLine.split("deadline ")[1].split(" /by ")[0];
+        String date = inputLine.split(" /by ")[1];
+        tasksList[taskCount] = new Deadline(description, date);
+        System.out.println("  Ok, I've added this task:");
+        System.out.println("  " + tasksList[taskCount]);
+        taskCount++;
+        System.out.printf("  Now you have %d tasks in the list. Anything else?\n", taskCount);
+        return taskCount;
+    }
+
+    public static int addToDo(Task[] tasksList, int taskCount, String inputLine) {
+        tasksList[taskCount] = new ToDo(inputLine.split("todo ")[1]);
+        System.out.println("  Ok, I've added this task:");
+        System.out.println("  " + tasksList[taskCount]);
+        taskCount++;
+        System.out.printf("  Now you have %d tasks in the list. Anything else?\n", taskCount);
+        return taskCount;
+    }
+
+    public static void printTasks(Task[] tasksList, int taskCount) {
+        if (taskCount == 0) {
+            System.out.println("  You haven't added any task yet.");
+        } else {
+            for (int i = 1; i < taskCount + 1; i++) {
+                System.out.println("  " + i + "." + tasksList[i - 1]);
+            }
+        }
+    }
+
+    public static void exitProgram() {
```

Comment:
Consider using consistent method names across similar methods. For example, since you named your welcome message method showWelcomeMessage(), your exit message method is recommended to be showExitMessage() so that it is more consistent and easier for other people to trace through your code.